modules/bullet/space_bullet.cpp

```diff
@@ -945,7 +945,7 @@ bool SpaceBullet::test_body_motion(RigidBodyBullet *p_body, const Transform &p_f
 
 	btVector3 motion;
 	G_TO_B(p_motion, motion);
-	if (!motion.fuzzyZero()) {
+	//if (!motion.fuzzyZero()) {
```

Comment:
Please remove code entirely instead of commenting it out. We use version control after all, we can find the old code back if we need it :slightly_smiling_face: